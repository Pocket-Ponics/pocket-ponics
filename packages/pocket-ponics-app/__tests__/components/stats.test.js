import React from 'react'
import { shallow } from 'enzyme'

import Stats from '../../components/greenhouse/stats'

describe('Stats Display', () => {
	test('renders correctly', () => {
		const tree = shallow(
			<Stats
				stats={{
					battery: 100,
					water: 80,
					nutrient: 40
				}}/>).html()
		expect(tree).toMatchSnapshot()
	})

	test('renders with low stats', () => {
		const tree = shallow(
			<Stats
				stats={{
					battery: 40,
					water: 20,
					nutrient: 0
				}}/>).html()
		expect(tree).toMatchSnapshot()
	})
})