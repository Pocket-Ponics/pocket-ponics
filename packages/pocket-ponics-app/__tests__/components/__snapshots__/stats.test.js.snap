// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stats Display renders correctly 1`] = `"<View style=\\"margin-top:20px\\"><Text style=\\"font-size:40px;color:#FFFFFF;text-align:center\\">Current Stats</Text><View style=\\"flex-direction:row;padding:10px\\"><Image source=\\"1\\" style=\\"height:50px;width:60px;resize-mode:contain\\"></Image><Text style=\\"font-size:25px;padding-left:30px;color:#FFFFFF;flex:1\\">Battery: 100%</Text></View><View style=\\"flex-direction:row;padding:10px\\"><Image source=\\"1\\" style=\\"height:50px;width:60px;resize-mode:contain\\"></Image><Text style=\\"font-size:25px;padding-left:30px;color:#FFFFFF;flex:1\\">Water: 80%</Text></View><View style=\\"flex-direction:row;padding:10px\\"><Image source=\\"1\\" style=\\"height:50px;width:60px;resize-mode:contain\\"></Image><Text style=\\"font-size:25px;padding-left:30px;color:#FFFFFF;flex:1\\">Nutrients:  40%</Text></View></View>"`;

exports[`Stats Display renders with low stats 1`] = `"<View style=\\"margin-top:20px\\"><Text style=\\"font-size:40px;color:#FFFFFF;text-align:center\\">Current Stats</Text><View style=\\"flex-direction:row;padding:10px\\"><Image source=\\"1\\" style=\\"height:50px;width:60px;resize-mode:contain\\"></Image><Text style=\\"font-size:25px;padding-left:30px;color:#FFFFFF;flex:1\\">Battery: 40%</Text></View><View style=\\"flex-direction:row;padding:10px\\"><Image source=\\"1\\" style=\\"height:50px;width:60px;resize-mode:contain\\"></Image><Text style=\\"font-size:25px;padding-left:30px;color:#FFFFFF;flex:1\\">Water: 20%</Text></View><View style=\\"flex-direction:row;padding:10px\\"><Image source=\\"1\\" style=\\"height:50px;width:60px;resize-mode:contain\\"></Image><Text style=\\"font-size:25px;padding-left:30px;color:#FFFFFF;flex:1\\">Nutrients:  0%</Text></View></View>"`;
